config/projects.yml

```diff
@@ -269,12 +269,12 @@ intl-bundle:
       php: ['7.1']
       versions:
         symfony: ['2.8', '3.1', '3.2']
-        sonata_user: ['2', '3']
+        sonata_user: ['3']
     2.x:
       php: ['5.3', '5.4', '5.5', '5.6', '7.0', '7.1']
       versions:
         symfony: ['2.3', '2.7', '2.8', '3.1', '3.2']
-        sonata_user: ['2', '3']
```

Comment:
> there is no reason to change this

I think there is a difference between allowing people to stay on 2.x and making sure tests pass for it. It means more maintenance for no good reason. IMO we should spend no time on it. Changing `composer.json` to explicitely disallow it is another discussion.